src/rabbit_mgmt_util.erl

```diff
@@ -810,7 +810,13 @@ decode(<<"">>) ->
     {ok, #{}};
 decode(Body) ->
     try
-        {ok, rabbit_json:decode(Body)}
+        %% handle double encoded JSON, see rabbitmq/rabbitmq-management#839
+        case rabbit_json:decode(Body) of
+            Val when is_map(Val)    -> {ok, Val};
+            Val when is_list(Val)   -> {ok, maps:from_list(Val)};
+            Bin when is_binary(Bin) -> {error, not_json};
```

Comment:
Hmm. As you said [here](https://github.com/rabbitmq/rabbitmq-management/issues/839#issuecomment-689239110) doubly-encoded JSON _is valid JSON_.

Can we return something other than `not_json`?

Reply:
See [this commit](https://github.com/rabbitmq/rabbitmq-management/pull/848/commits/cd8ed9f97a5ca0634c4f3cd23e1842641b6e03be). Feel free to rebase and drop it if you'd like, however!

Reply:
We can indicate that the payload was invalid (not invalid JSON per se). Discussed on Slack.